src/parameterizations/lateral/MOM_tidal_forcing.F90

```diff
@@ -43,6 +49,9 @@ module MOM_tidal_forcing
   integer :: struct(MAX_CONSTITUENTS) !< An encoded spatial structure for each constituent
   character (len=16) :: const_name(MAX_CONSTITUENTS) !< The name of each constituent
 
+  real :: time_ref !< Reference time (t = 0) used to calculate tidal forcing.
```

Comment:
This time_ref should stored as a time_type, rather than as a real number. Also, all real variables are supposed to have their units documented in comments. (This is not true for time_types, which already have unambiguous units.